src/test/index.spec.js

```diff
@@ -0,0 +1,31 @@
+import { app, chai, expect } from './testHelpers/config.spec';
+
+describe('Index js', () => {
+  it('should display a welcome message successfully', (done) => {
+    chai
+      .request(app)
+      .get('/')
+      .end((err, res) => {
+        expect(res.status).to.be.equal(200);
+        expect(res.body.status).to.be.equal('success');
+        expect(res.body.data).to.be.an('object')
+        expect(res.body.data.message).to.be.equal('Welcome to Barefoot Nomad');
```

Comment:
can we use the messages object we created in the utils folder here?

Reply:
yes, done